res/schema.xml

```diff
@@ -426,4 +426,21 @@ METADATA
       ALTER TABLE cues ADD COLUMN color INTEGER DEFAULT 4294901760 NOT NULL;
     </sql>
   </revision>
+  <revision version="28" min_compatible="3">
+    <description>
+      Add traktor color support.
+      See library/traktor/traktorfeature.cpp
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS traktor_cues (
```

Comment:
Please add a comment to the code, how the table will be used.

Reply:
I am descriptive in the file now.
Don't know if this is enough.